pkgs/applications/science/biology/mrbayes/default.nix

```diff
@@ -1,18 +1,18 @@
-{lib, stdenv, fetchurl, readline}:
+{ lib, stdenv, fetchFromGitHub, readline }:
 
 stdenv.mkDerivation rec {
-  # FIXME: replace Makefile so we can build MPI & MAC support
+  pname = "mrbayes";
+  version = "3.2.7";
```

Comment:
According to https://nbisweden.github.io/MrBayes/download.html the latest is 3.2.7a

```suggestion
  version = "3.2.7a";
```

Reply:
Why has this been resolved and merged without addressing the issue?

Reply:
there is not such release on github. assumed this was a typo.

Reply:
basic sorting is hard...

![image](https://user-images.githubusercontent.com/7258858/127153668-b4afb15e-b52e-469a-b04d-e559cc48a070.png)